backend/typescript/services/implementations/reviewService.ts

```diff
@@ -271,30 +271,38 @@ class ReviewService implements IReviewService {
         );
 
         // Delete tags (if necessary)
-        const deleteTags = Promise.all(
-          reviewToDelete.tags.map((tag: PgTag) => {
-            return PgReviewTag.findAll({
-              where: { tag_id: tag.id },
-            }).then(async (ret: PgReviewTag[]) => {
-              if (ret.length === 1) {
-                // Delete tags
-                await PgTag.destroy({
-                  where: { id: [tag.id] },
-                });
-              }
+        // const deleteTags = Promise.all(
```

Comment:
is this temporarily commented out to be fixed later? or can we clean up the code?